scripts/create_perf_json.py

```diff
@@ -1582,6 +1585,30 @@ def append_to_desc(s: str):
 
         return jo
 
+    def count_counters(self, event_type, pmon_events):
+        # Count number of counters in each PMU unit
```

Comment:
Docstrings should be documented with triple quotes.
```
"""Count number of counters in each PMU unit."""
```

Reply:
Updated